src/widgets/ProgressDialog.cpp

```diff
@@ -1047,6 +1049,13 @@ ProgressDialog::~ProgressDialog()
    if (mHadFocus && SearchForWindow(wxTopLevelWindows, mHadFocus)) {
       mHadFocus->SetFocus();
    }
+
+   using namespace std::chrono;
+   wxLogInfo(
+      "Operation '%s' took in %f seconds. Poll was called %d times and took %f seconds. Yield was called %d times.",
```

Comment:
Either say "took" or "finished in"

Reply:
Shoot, my bad, changed the text sometime in the middle xD

It was `completed` initially, by I felt that this is not quite right, because this message will be shown for cancelled tasks as well